aiconfig-docs/docs/overview/model-parsers.md

```diff
@@ -4,20 +4,27 @@ sidebar_position: 10
 
 # Supported Models
 
-`aiconfig` is designed to be model-agnostic and multi-modal, allowing it to work with any model from any provider. For more information on how this works, please see [Extensibility](/docs/extensibility)
-
-## Default Models
-
-`aiconfig` ships with out-of-the-box support for a limited number of models.
+| Provider | Model | Language | Support |  
+| --- | --- | ---| --- | 
+| OpenAI | **GPT3.5** | Python | ✅ Built-in | 
+| OpenAI | **GPT3.5** | Typescript | ✅ Built-in |
+| OpenAI | **GPT4** | Python | ✅ Built-in |
+| OpenAI | **GPT4** | Typescript | ✅ Built-in |
+| OpenAI | **Dall-E 3** | Python | ✅ Built-in |
+| HuggingFace Inference Endpoints | **Text Generation** | Python | ✅ Built-in |
+| HuggingFace Inference Endpoints | **Text Generation** | Typescript | ✅ Built-in |
+| Google | **PaLM 2** | Python | ✅ Built-in |
+| Google | **PaLM 2** | Typescript | ✅ Built-in |
+| Google | **Gemini** | Python | ✅ Built-in |
+| Meta | **Llama 2** | Python | 🤝 [Extension](https://github.com/lastmile-ai/aiconfig/tree/main/extensions/llama/python) |
+| Meta | **Llama 2** | Typescript | 🤝 [Extension](https://github.com/lastmile-ai/aiconfig/tree/main/extensions/llama/typescript) |
+| Meta | **Llama Guard** | Python | 🤝 [Extension](https://github.com/lastmile-ai/aiconfig/tree/main/extensions/llama/typescript) |
+| HuggingFace Transformer Pipelines | **Text Generation** | Python | 🤝 [Extension](https://github.com/lastmile-ai/aiconfig/tree/main/extensions/HuggingFaceTransformers/python) |
 
 :::info
-We plan on keeping core `aiconfig` lightweight so it doesn't become a monolithic project. Instead, AIConfig extensions should be created and shared to support additional models and functionality. See
+We plan on keeping core `aiconfig` lightweight so it doesn't become a monolithic project. We recommend using AIConfig extensions to expand support for additional models and functionality.
 :::
 
-### OpenAI
-
-### Google
-
-### Hugging Face
+### Build your own Model Parser
```

Comment:
This can just link to extensibility page

Reply:
Added a link to the extensibility page